docs/modules/ROOT/pages/how-tos/configuring/creating-secrets.adoc

```diff
@@ -2,18 +2,23 @@
 
 When you build your pipeline, you might want to add tasks that require **secrets** in order to access external resources.
 
-NOTE: One such task is the link:https://github.com/konflux-ci/build-definitions/tree/main/task/sast-snyk-check[sast-snyk-check] task that uses the third-party service link:https://snyk.io/[snyk] to perform static application security testing (SAST) as a part of the default {ProductName} pipeline. Use this procedure to upload your snyk.io token. Name the secret `sast_snyk_task` so that the snyk task in the {ProductName} pipeline will recognize it and use it.
+.Secrets fall into three catagories of when they can be added:
```

Comment:
1. Usually we don't mark such introductory sentences as titles (with . at the beginning), even if they serve as list titles. This sentence can remain a plain text.

2. Writers are discouraged from using the passive voice where it can be avoided. The preferred wording would be: ...of when you add them. Or even simpler: There're three categories of secrets depending on when you add them.

3. catagories > categories.